src/lib/lager.js

```diff
@@ -4,32 +4,38 @@ const Promise = require('bluebird');
 const _ = require('lodash');
 const program = require('commander');
 const inquirer = require('inquirer');
+const icli = require('./icli');
 
 
-// Configure error reporting for dev environment
-// @TODO use bunyan for logs, including errors
-const PrettyError = require('pretty-error');
-const pe = new PrettyError();
+if (process.env.NODE_ENV === 'development') {
+  // Configure error reporting for dev environment
+  // @TODO use bunyan for logs, including errors
+  const PrettyError = require('pretty-error');
+  const pe = new PrettyError();
 
-// To render exceptions thrown in non-promies code:
-process.on('uncaughtException', e => {
-   console.log(pe.render(e));
-});
+  // To render exceptions thrown in non-promies code:
+  process.on('uncaughtException', e => {
+     console.log('Uncaught exception');
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 4 space characters but found 5.](https://www.codacy.com/app/lager/lager/file/2779185919/issues/source?bid=3360491&fileBranchId=3365065#l18)